demos/segmentation_demo/segmentation_demo.h

```diff
@@ -9,54 +9,24 @@
 #include <gflags/gflags.h>
 #include <iostream>
 
-/// @brief message for help argument
 static const char help_message[] = "Print a usage message.";
-
-/// @brief message for images argument
 static const char image_message[] = "Required. Path to an .bmp image.";
-
-/// @brief message for model argument
 static const char model_message[] = "Required. Path to an .xml file with a trained model.";\
-
-/// @brief message for assigning cnn calculation to device
 static const char target_device_message[] = "Optional. Specify the target device to infer on (the list of available devices is shown below). " \
-"Default value is CPU. Use \"-d HETERO:<comma-separated_devices_list>\" format to specify HETERO plugin. " \
-"The demo will look for a suitable plugin for a specified device.";
-
-/// @brief message for clDNN custom kernels desc
+                                            "Default value is CPU. Use \"-d HETERO:<comma-separated_devices_list>\" format to specify HETERO plugin. " \
+                                            "The demo will look for a suitable plugin for a specified device.";
 static const char custom_cldnn_message[] = "Required for GPU custom kernels. "\
-"Absolute path to the .xml file with the kernels descriptions.";
-
-/// @brief message for user library argument
+                                           "Absolute path to the .xml file with the kernels descriptions.";
 static const char custom_cpu_library_message[] = "Required for CPU custom layers. " \
-"Absolute path to a shared library with the kernels implementations.";
-
-/// @brief message for config argument
-static constexpr char config_message[] = "Path to the configuration file. Default value: \"config\".";
+                                                 "Absolute path to a shared library with the kernels implementations.";
+static const char config_message[] = "Path to the configuration file. Default value: \"config\".";
```

Comment:
Why not replace all `const` with `constexpr` instead? :)

Reply:
Because it would make no difference and the diff would be longer.